sympy/assumptions/handlers/matrices.py

```diff
@@ -405,6 +405,38 @@ def DiagonalMatrix(expr, assumptions):
     Factorization = staticmethod(partial(_Factorization, Q.diagonal))
 
 
+class AskHermitianHandler(CommonHandler):
+
+    Identity, ZeroMatrix = [staticmethod(CommonHandler.AlwaysTrue)]*2
+
+    @staticmethod
+    def MatrixExpr(expr, assumptions):
+        if not expr.is_square:
+            return False
+        for i in range(expr.rows):
```

Comment:
I wrote the logic for `hermitian` instead of using `is_hermitian` property. Is it okay?
I was trying to use `is_hermitian` property at first, but somehow it was just return `None` instead of `True/False`.